### PR TITLE
Added creation of a release and tender.status = planned #47

### LINKED ISSUE #47: XIV:1: Add instruction about OCDS release
> Where a procuring entity intends to conduct a covered procurement using an electronic auction, the entity shall provide each participant, before commencing the electronic auction, with:

This can be made in a second release under the same OCID. We should add/update instructions to do so.

### PR DESCRIPTION
**Pull request by ColinMaudry**
I wasn't which tag should be added to the release. On the other hand, in GPA guidance, releases are sometimes created without a tag.

Closes #47

### CODE REVIEW

**Comment by ColinMaudry** [comment]
When you're back I'd like to have a chat with you about all those CI runs that fail :)

**Comment by duncandewhurst** [comment]
The lint check is failing because `schema/profile/release-schema.json` is missing validation properties.

You should be able to run OCDS Kit's [schema-strict](https://ocdskit.readthedocs.io/en/latest/cli/schema.html#schema-strict) command to fix that.

You didn't change the schema in this PR though, so I checked the standard maintenance scripts repo and it looks like @jpmckinney added this test 5 days ago in https://github.com/open-contracting/standard-maintenance-scripts/commit/c420fcb17627c8526c78c28acdde81aa70ff42ea, so a failure doesn't always relate to something you've done as the tests can be a moving target.

File: docs/gpa.md [hunk 1]
```diff
@@ -588,6 +588,8 @@ This requires the [Participation Fees](https://github.com/open-contracting-exten
         <td>the automatic evaluation method, including the mathematical formula, that is based on the evaluation criteria set out in the tender documentation and that will be used in the automatic ranking or re-ranking during the auction;</td>
         <td markdown=1>
 
+1. [Create an OCDS release](../operations/#create-a-release)
+1. Set `tender/status` to 'planned'
```
Comment (jpmckinney) [inline docs/gpa.md:592]: Why set it to 'planned'?

**Comment by jpmckinney** [comment]
Please check my changes in 71fbdec

File: docs/gpa.md [hunk 2]
```diff
@@ -376,6 +378,8 @@ If you are interested in using OCDS to publish this, please contact <data@open-c
         <td markdown=1>
 
 1. [Create an OCDS release](../operations/#create-a-release)
+1. Add 'tenderUpdate' to the `tag` array
```
Comment (ColinMaudry) [inline docs/gpa.md:381]: At that point, have they necessarily published tender data? Is it possible they haven't published a notice of intended procurement and the related OCDS, and that they publish information about this process for the first time?

Comment (jpmckinney) [reply docs/gpa.md:381]: I'm not sure such a scenario is possible: publishing tender documentation without any prior notice of the tender.

I guess there's a possibility when "turning on" their OCDS publication, that they need to refer to old processes published as non-OCDS (which for whatever reason they don't want to re-publish as OCDS), but I think that scenario is very narrow.

File: docs/gpa.md (hunk 1 above)
Comment (ColinMaudry) [reply docs/gpa.md:598]: I don't remember... I guess it should be 'active', since it's about how the electronic auction is setup.

**Comment by ColinMaudry** [comment]
The test fails at:

```bash
echo "Copy the built files to the remote server..."
rsync -avz --delete-after build/ ocds-docs@standard.open-contracting.org:web/staging/$PATH_PREFIX${GITHUB_REF##*/}
```

with 

```
 Host key verification failed.
rsync: connection unexpectedly closed (0 bytes received so far) [sender]
rsync error: unexplained error (code 255) at io.c(235) [sender=3.1.2]
Error: Process completed with exit code 255.
```

**Comment by jpmckinney** [comment]
I'll try re-running the job. It might be related to port knocking that we introduced to the server.

File: docs/gpa.md (hunk 1 above)
Comment (jpmckinney) [reply docs/gpa.md]: Let's just delete this instruction, since it should already be 'active' in earlier releases.

File: docs/gpa.md [hunk 3]
```diff
@@ -378,7 +378,8 @@ If you are interested in using OCDS to publish this, please contact <data@open-c
         <td markdown=1>
 
 1. [Create an OCDS release](../operations/#create-a-release)
-1. Add 'tenderUpdate' to the `tag` array
+1. Set the `tag` array:
+  1. If data has been published acc
```
Comment (jpmckinney) [inline docs/gpa.md:382]: This seems incomplete?

File: docs/gpa.md [hunk 4]
```diff
@@ -568,7 +569,7 @@ This requires the [Participation Fees](https://github.com/open-contracting-exten
   </table>
 </div>
 
-## [Article XIV](https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm#articleXIV) Electronic Auctions
+  ## [Article XIV](https://www.wto.org/english/docs_e/legal_e/rev-gpr-94_01_e.htm#articleXIV) Electronic Auctions
```
Comment (jpmckinney) [inline docs/gpa.md:572]: Why add whitespace here? I think it breaks the Markdown.

**Review by jpmckinney — COMMENTED** [review]
I've emailed Dogsbody about the build failure, but there are a couple issues with the PR.

File: docs/gpa.md (hunk 3 above)
Comment (ColinMaudry) [reply docs/gpa.md:382]: I had tried to write guidance to cover all the scenarios described in my comment: https://github.com/open-contracting-extensions/government-procurement-agreement/pull/51#discussion_r524402113 but forgot to remove it when I commited.

File: docs/gpa.md (hunk 4 above)
Comment (ColinMaudry) [reply docs/gpa.md:572]: Fixed.

**Pull request merged**